packages/openapi3/src/cli/actions/convert/generators/generate-types.ts

```diff
@@ -47,8 +48,7 @@ function getTypeFromSchema(schema: OpenAPI3Schema): string {
 
 export function getRefName(ref: string): string {
   const name = ref.split("/").pop() ?? "";
-  // TODO: account for `.` in the name
-  return name;
+  return name.split(".").map(printIdentifier).join(".");
```

Comment:
Waiting for the playground to see what this does but does this mean if you have a re like `#/components/schemas//foo.bar` it will do `foo.bar`

are the `.` already used to group under namespaces?